prow/config/secret/secret_test.go

```diff
@@ -0,0 +1,48 @@
+package secret
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+func TestLoadSingleSecret(t *testing.T) {
+	tests := []struct {
+		name    string
+		content string
+		want    []byte
+		wantErr bool
```

Comment:
Please do not use booleans for this but a concrete error or error message, otherwise the tests might start rotting when the Loading fails due to a different reason